utils/land-patterns.stanza

```diff
@@ -23,6 +23,94 @@ public defn ref-label (x:Double, y:Double) :
 public defn ref-label () :
   ref-label(0.0, 0.0)
 
+; helper to compute the shape of an orientation marker
+defn compute-orientation-marker (
+  lp: LandPattern|Self, ; the landpattern, or Self object  
+  p1: Ref,               ; the reference corresponding to pin 1, which may differ for various components (p[1], A[1], etc)
+  angle:Double|Anchor,  ; the anchor, which controls the angle of the shape 
+  distance:Double       ; the distance from the pin to draw
+  shape: Shape          ; the shape of the indicator
+) -> Shape:
+  val angle* = 
+    match(angle):
+      (anchor:Anchor):
+          switch(anchor):
+            C:  0.0  * PI
+            NE: 0.25 * PI
+            N:  0.5  * PI
+            NW: 0.75 * PI
+            W:  1.0  * PI
+            SW: 1.25 * PI
+            S:  1.5  * PI
+            SE: 1.75 * PI
+            E:  0.0
```

Comment:
Same value than for `C`. Is it ok?

Reply:
Yes. `C` is an outlier, but I think it's better to be weird than to crash

Reply:
Yes, it is always tricky to handle it properly